--- origsrc/kjewishdatefinder-1.1/kjewishdatefinder.pro	2006-03-20 00:32:15.000000000 -0600
+++ src/kjewishdatefinder-1.1/kjewishdatefinder.pro	2008-05-19 22:24:16.250000000 -0500
@@ -3,8 +3,8 @@
 
 CONFIG += qt release
 
-INCLUDEPATH += $(KDEDIR)/include
-LIBS += -L$(KDEDIR)/lib -lkdeui
+INCLUDEPATH += /usr/include/kde3
+LIBS += -L/usr/lib/kde3/devel -lkdeui -lkdecore
 
 QMAKE_CXXFLAGS_RELEASE += `pkg-config libhdate --cflags`
 LIBS += `pkg-config libhdate --libs`
